src/EditorFeatures/Core/Implementation/IntelliSense/Helpers.cs

```diff
@@ -72,7 +140,60 @@ internal static IEnumerable<object> BuildClassifiedTextElements(ImmutableArray<T
             return paragraphs;
         }
 
-        internal static object CreateParagraphFromLines(IReadOnlyList<ClassifiedTextElement> lines)
+        private static void NavigateToQuickInfoTarget(string navigationTarget, Document document, IEnumerable<Lazy<IStreamingFindUsagesPresenter>> streamingPresenters)
+        {
+            var navigateToLinkService = document.Project.Solution.Workspace.Services.GetRequiredService<INavigateToLinkService>();
+            if (Uri.TryCreate(navigationTarget, UriKind.Absolute, out var absoluteUri))
+            {
+                navigateToLinkService.TryNavigateToLinkAsync(absoluteUri, CancellationToken.None);
+                return;
+            }
+
+            SymbolKeyResolution resolvedSymbolKey;
+            try
+            {
+                resolvedSymbolKey = SymbolKey.Resolve(navigationTarget, document.Project.GetCompilationAsync(CancellationToken.None).WaitAndGetResult(CancellationToken.None), cancellationToken: CancellationToken.None);
+            }
+            catch
+            {
+                // Ignore symbol resolution failures. It likely is just a badly formed URI.
+                return;
+            }
+
+            if (resolvedSymbolKey.GetAnySymbol() is { } symbol)
+            {
+                GoToDefinitionHelpers.TryGoToDefinition(symbol, document.Project, streamingPresenters, CancellationToken.None);
+                return;
+            }
+        }
+
+        private static ClassifiedTextRunStyle GetClassifiedTextRunStyle(TaggedTextStyle style)
+        {
+            var result = ClassifiedTextRunStyle.Plain;
+            if ((style & TaggedTextStyle.Emphasis) == TaggedTextStyle.Emphasis)
+            {
+                result |= ClassifiedTextRunStyle.Italic;
+            }
+
+            if ((style & TaggedTextStyle.Strong) == TaggedTextStyle.Strong)
+            {
+                result |= ClassifiedTextRunStyle.Bold;
+            }
+
+            if ((style & TaggedTextStyle.Underline) == TaggedTextStyle.Underline)
+            {
+                result |= ClassifiedTextRunStyle.Underline;
+            }
+
+            if ((style & TaggedTextStyle.Code) == TaggedTextStyle.Code)
+            {
+                result |= ClassifiedTextRunStyle.UseClassificationFont;
```

Comment:
what does `UseClassificationFont` mean?

Reply:
This is an editor API. Uses the font assigned to the classification type in the IDE, which defaults to a fixed-width font but could be changed.